packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/ApprovalRequestsTable.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { useState } from 'react';
+
+import { components } from '@/api';
+import { ScopedCssBaselineContainerClassName } from '@/components/ContainerCssBaseline';
+import { useMoniteContext } from '@/core/context/MoniteContext';
+import { MoniteScopedProviders } from '@/core/context/MoniteScopedProviders';
+import { useCurrencies } from '@/core/hooks';
+import { useEntityUserByAuthToken } from '@/core/queries';
+import { useIsActionAllowed } from '@/core/queries/usePermissions';
+import { AccessRestriction } from '@/ui/accessRestriction';
+import { CounterpartCell } from '@/ui/CounterpartCell';
+import { LoadingPage } from '@/ui/loadingPage';
+import {
+  TablePagination,
+  useTablePaginationThemeDefaultPageSize,
+} from '@/ui/table/TablePagination';
+import { DateTimeFormatOptions } from '@/utils/DateTimeFormatOptions';
+import { ActionEnum } from '@/utils/types';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { Box, Stack } from '@mui/material';
+import { DataGrid, GridValueFormatterParams } from '@mui/x-data-grid';
+
+import { addDays, formatISO } from 'date-fns';
+
+import {
+  ApprovalRequestStatus,
+  FILTER_TYPE_STATUS,
+  FILTER_TYPE_CREATED_AT,
+  FILTER_TYPE_ADDED_BY,
+  FILTER_TYPE_CURRENT_USER,
+} from '../consts';
+import { FilterTypes, FilterValue } from '../types';
+import { ApprovalRequestStatusChip } from './ApprovalRequestStatusChip';
+import { ApproveButton } from './ApproveButton';
+import { Filters as FiltersComponent } from './Filters';
+import { RejectButton } from './RejectButton';
+import { UserCell } from './UserCell';
+
+interface ApprovalRequestsTableProps {
+  /**
+   * Triggered when the filtering options are changed
+   *
+   * @param filter - An object containing the filter parameters.
+   * @param filter.field - The field to filter by, specified as a keyof FilterTypes.
+   * @param filter.value - The value to be applied to the filter, of type FilterValue.
+   */
+  onChangeFilter?: (filter: {
```

Comment:
We discussed with @nekedos  that `onChangeFilter` callback is unnecessary, and we will remove it from other tables in the future. 

On the other hand, the familiar `onRowClick` is missing for some reason. It seems that the table is useless without it.

Reply:
I removed `onChangeFilter` and added `onRowClick`